apps/condo/domains/organization/components/DeleteEmployeeButtonWithReassignmentModel.tsx

```diff
@@ -0,0 +1,245 @@
+                </Typography.Text>
+                {updatedTicketsCount !== null  && <Typography.Text>
+                    {intl.formatMessage({ id: 'employee.reassignTickets.notification.progress' }, { activeTicketsOrganizationEmployeeCount, updatedTicketsCount })}
+                </Typography.Text>}
+            </Space>
+        )
+    }, [activeTicketsOrganizationEmployeeCount, intl])
+
+    const handleDeleteButtonClick = () => {
+        setIsDeleting(true)
+        setIsConfirmVisible(false)
+
+        runMutation(
+            {
+                action: softDeleteAction,
+                onError: (e) => { throw e },
+                OnErrorMsg: () => ({
+                    message: notificationTitle(NotificationTitleErrorLabel),
+                    description: notificationMessage(NotificationMessageErrorLabel),
+                    duration: 0,
+                    key: 'deleteOrganizationEmployee',
+                }),
+                OnCompletedMsg: () => ({
+                    message: notificationTitle(NotificationTitleErrorLabel),
+                    description: notificationMessage(NotificationMessageDeleteUserLabel),
+                    key: 'deleteOrganizationEmployee',
+                }),
+                onFinally: () => {
+                    setIsDeleting(false)
+                },
+                intl,
+            },
+        )
+    }
+    
+    const handleDeleteAndReassignTicketsClick = async () => {
+        setIsDeleting(true)
+        setIsConfirmVisible(false)
+
+        let updatedTicketsCount = 0
+
+        while (updatedTicketsCount < activeTicketsOrganizationEmployeeCount) {
+            notificationApi.warning({
+                message: notificationTitle(NotificationTitleWarningLabel),
+                description: notificationMessage(NotificationMessageWarningLabel, updatedTicketsCount),
+                duration: 0,
+                key: 'reassignTicket',
+            })
+
+            try {
+                const { data: ticketsToReassign } = await loadTicketsToReassign()
+
+                if (isEmpty(ticketsToReassign?.tickets)) break
+                const { data: reassignedTickets }  = await updateTickets({
+                    variables: {
+                        data: ticketsToReassign?.tickets?.map(ticket => ({
+                            id: ticket?.id,
+                            data: getTicketReassignData(ticket),
+                        })),
+                    },
+                })
+
+                updatedTicketsCount += reassignedTickets?.tickets?.length
+                await sleep(1000)
+            } catch (err) {
+                if (errorLoadTickets || errorUpdateTickets || err) {
+                    notificationApi.error({
+                        message: notificationTitle(NotificationTitleErrorLabel),
+                        description: notificationMessage(NotificationMessageErrorLabel),
+                        duration: 0,
```

Comment:
Why duration is 0 here? (just asking)

Reply:
I thought it took the user more than 5 seconds to read the error